src/main/java/pl/isa/javaers/ExchangeRateHistoryMenu.java

```diff
@@ -0,0 +1,41 @@
+package pl.isa.javaers;
+
+import java.util.Scanner;
+
+public class ExchangeRateHistoryMenu {
+    private String[] exchangeRateHistoryMenu;
+
+    public ExchangeRateHistoryMenu() {
+        this.exchangeRateHistoryMenu = new String[]{"Back to main menu", "Show exchange rate from chosen dates."};
+    }
+    public void display() {
```

Comment:
Jeżeli metoda display powtarza się w różnych klasach menu, to dobrze byłoby stworzyć jakąś klasę bazową, która by zawierała tą metodę display.